graviti/openapi/file.py

```diff
@@ -43,7 +43,9 @@ def _request_upload_permission(
         is_internal=config.is_internal,
         expired=_EXPIRED_IN_SECOND,
     )
-    permission["expire_at"] = datetime.fromisoformat(permission.pop("expire_at").rstrip("Z"))
+    permission["expire_at"] = datetime.fromisoformat(
+        permission.pop("expire_at").replace("Z", "+00:00")
```

Comment:
replace look very slow, do we have faster ways?

Reply:
So far not found.